pkg/operator/nfs/spec.go

```diff
@@ -89,7 +93,7 @@ func newStatefulSetForNFSServer(cr *nfsv1alpha1.NFSServer) *appsv1.StatefulSet {
 					Containers: []corev1.Container{
 						{
 							Name:  "nfs-server",
-							Image: "rook/nfs:master",
```

Comment:
Oh wow, that bug must have been there for a long time. We should definitely not be setting this to the master tag!